docs/snippets/azkv-external-secret.yaml

```diff
@@ -35,3 +35,12 @@ spec:
     remoteRef:
       key: key/dev-key-test
 
+  # type/name/tagname of the tag in the secret in Azure KV
+  - secretKey: dev-tag-test
+    remoteRef:
+      key: tag/dev-tag-test/tagname
```

Comment:
Hello @gusfcarvalho thanks for the input, I like the simplicity of this way but I agree with property it may be clearer.
I'll change it.

Reply:
The prefix `secret/key/cert` refers to keyvault object types: https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates#object-types

We shouldn't add `tags` as prefix because each object type (secret/key/cert) can have it's own set of tags.